ksqldb-serde/src/main/java/io/confluent/ksql/logging/processing/LoggingSerdeUtil.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.logging.processing;
+
+import io.confluent.ksql.util.ErrorMessageUtil;
+import java.util.List;
+
+final class LoggingSerdeUtil {
+
+  private LoggingSerdeUtil() {
+  }
+
+  static List<String> getCause(final Throwable exception) {
+    final List<String> cause = ErrorMessageUtil.getErrorMessages(exception);
+    cause.remove(0);
```

Comment:
I know this is a cut & paste, but I've seen this `remove(0)` remove the actual cause of the error, e.g. I've seen cases where `exception` is a simple `NullPointerException` and the `remove(0)` has resulted in an empty `cause` list.

Personally, I'd remove the `remove` and inline this method. 

Which would only leave `getRecordComponent`, which I'd also be tempted to remove, and just add appropriate `"key"` and `"value"` constants to `ProcessingLogMessageSchema` instead.

Reply:
The first error message (the one being removed by this line) is already captured in the `errorMessage` field, which is why it's purposefully left out of the `cause` field. It sounds like you're advocating for duplication of the initial error message, so that the `cause` field is complete?